extensions/native/guest-macro/src/lib.rs

```diff
@@ -0,0 +1,23 @@
+use p3_baby_bear::BabyBear;
+
+use crate::parse_compiler_output::CompiledKernel;
+
+mod parse_compiler_output;
+mod parse_kernel;
+mod transportation;
+const UTILITY_CELL: usize = 0;
```

Comment:
this is not safe: you are making an assumption that ptr=0 is free to use, whereas we might be using it for another purpose. I think you should use `A0` from the native-compiler crate?